units/en/unit1/rl-framework.mdx

```diff
@@ -61,8 +61,6 @@ In a chess game, we have access to the whole board information, so we receive a
 <figcaption>In Super Mario Bros, we only see the part of the level close to the player, so we receive an observation.</figcaption>
 </figure>
 
-In Super Mario Bros, we only see the part of the level close to the player, so we receive an observation.

```

Comment:
This repetition is intended, one is the text the other the figcaption (and given most people don't read figcaption 😆 )